examples/recipes/pagination/queries/first.json

```diff
@@ -0,0 +1,12 @@
+{
+    "order": [
+        [
+            "Orders.number",
+            "asc"
+        ]
+    ],
+    "dimensions": [
+        "Orders.number"
+    ],
+    "limit": 5
+}
```

Comment:
@rchkv Subjectively, 2 spaces per tab look better for JSOIN 😄

Reply:
FYI I'd just run Prettier over everything - there's [a config file in the docs](https://github.com/cube-js/cube.js/blob/master/docs/.prettierrc) you can point Prettier at to get the same formatting rules as the docs